app/services/jbpm_process_service.rb

```diff
@@ -21,8 +21,8 @@ def signal(decision)
       bpm.signal_process_instance(options['container_id'], request.process_ref, options['signal_name'], :body => signal_options(decision))
     end
   rescue => err
-    ActionCreateService.new(request.id).create(:operation => Action::ERROR_OPERATION, :processed_by => 'system', :comments => err.message)
-    raise
+    # Signaling PAM is the last step after the approval has finished. Ignore the error if signal fails
+    Rails.logger.error("Failed to signal PAM with error: #{err.message}")
```

Comment:
Since I'm not as well versed in approval, this might be a stupid question, but basically is your comment here implying that is why line 24 is no longer needed?

Reply:
yes